Formula/haxe.rb

```diff
@@ -32,7 +32,8 @@ def install
       ENV["OPAMROOT"] = buildpath/"opamroot"
       ENV["OPAMYES"] = "1"
       system "opam", "init", "--no-setup"
-      system "opam", "install", "ocamlfind"
+      system "opam", "switch", "4.04.2"
```

Comment:
It seems weird to me that the stable version works with 4.05.0 but that HEAD doesn't. That's a pretty serious regression, which needs to be reported and fixed upstream.